frontend/root_words_ui/components/Form.js

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {InputLabel, Button, Box} from '@mui/material/';
+import MenuItem from '@mui/material/MenuItem';
+import FormHelperText from '@mui/material/FormHelperText';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+
+export default function Form() {
```

Comment:
Just remember that the rendering of the verse list will depend on the change in chapter choice. That can likely be handled in another issue though.